nav2_util/CMakeLists.txt

```diff
@@ -87,6 +87,10 @@ install(DIRECTORY include/
 
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
+  # the following line skips the linter which checks for copyrights
+  # remove the line when a copyright and license is present in all source files
```

Comment:
Same as my comment on the other PR. Wouldn't it be more accurate to say, "remove the line when all license headers are supported by the copyright checker tool and the linter tests pass," or something like that?